config/jobs/kubernetes-sigs/cluster-capacity/cluster-capacity-presubmits-release-1.21.yaml

```diff
@@ -2,6 +2,7 @@
 presubmits:
   kubernetes-sigs/cluster-capacity:
   - name: pull-cluster-capacity-verify-gofmt-release-1-21
```

Comment:
You should probably have a conversation with Cluster Capacity maintainers about pruning 1.23(potentially 1.24) and older jobs from prow.

Reply:
@ingvagabund @damemi, would this also be a good opportunity to add 1.27 jobs along with pruning?

Reply:
+1 for pruning and adding the 1.27 job